FusionIIIT/Fusion/urls.py

```diff
@@ -22,14 +22,8 @@
 from django.contrib import admin
 from django.contrib.auth import views as auth_views
 
-from rest_framework.authtoken.views import obtain_auth_token 
 
-
-
-urlpatterns = [
-
-    url(r'^api-token-auth/', obtain_auth_token, name='api_token_auth'),  
-                   
```

Comment:
why have you removed the token? Please elaborate

Reply:
I was using it at the time of api's testing. Well there is no problem in keeping it.

Reply:
Whatever changes you are doing while testing don't push them, only push the necessary changes related to the PR